gitpod-io/gitpod
# [dashboard] use workspace class component

## Description

Use the dropdown component for workspace classes in the project settings.

This also includes a fix for a bug that was introduced yesterday, where the dependency array would contain the query object rather than the actual query data, which causes many rerenders

<img width="989" alt="Screenshot 2023-04-13 at 15 21 34" src="https://user-images.githubusercontent.com/372735/231771822-8068f0ae-a8ae-41a7-853f-61149209836a.png">

## Related Issue(s)

Fixes WEB-126

## How to test

Create a project and check the settings

https://svenefftin994185c645.preview.gitpod-dev.com/orgs/join?inviteId=8bfb9176-45d0-4f34-a535-1c0a1072424a

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
